argo-workflows/ironic-to-nautobot-sync/workflowtemplates/sync.yaml

```diff
@@ -12,13 +12,11 @@ spec:
       container:
         image: ghcr.io/rackerlabs/understack/ironic-nautobot-client:latest
         command:
-          - nautobot-update-cf
+          - synchronize-provision-state-to-nautobot
```

Comment:
here it's `synchronize-provision-state-to-nautobot` but....

Reply:
good catch, it's supposed to be `synchronize-provision-state`, will fix.